README.md

```diff
@@ -23,13 +23,10 @@ To enable this DHCP provider, edit `/etc/foreman-proxy/settings.d/dhcp.yml` and
 
 Configuration options for this plugin are in `/etc/foreman-proxy/settings.d/dhcp_infoblox.yml` and include:
 
-* infoblox_user: API Username
-* infoblox_pw: API Password
+* username: API Username
+* passowrd: API Password
 * record_type: host / fixed_address (see different record types chapter)
```

Comment:
we validate `fixedaddress` while here and in setting example, we use term `fixed_address`